openproductapiclient/models.py

```diff
@@ -0,0 +1,47 @@
+import logging
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from solo.models import SingletonModel
+
+from .client import Client
+
+logger = logging.getLogger(__name__)
+
+
+class Configuration(SingletonModel):
```

Comment:
You'll want to be more specific to avoid confusion for downstream consumers:

```suggestion
class OpenProductConfiguration(SingletonModel):
```